Sources/LiveViewNative/Modifiers/View Configuration Modifiers/PreferredColorSchemeModifier.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  PreferredColorSchemeModifier.swift
+// LiveViewNative
+//
+//  Created by May Matyi on 4/24/23.
+//
+
+import SwiftUI
+
+/// Overrides the preferred color scheme for the presentation.
```

Comment:
I'd add a clarification that this modifier propagates _up_ to the nearest presentation container, since it's different than most modifiers that only alter their content.

Reply:
@shadowfacts I added a clarification. Feel free to suggest a change if you think it could be worded better.